geode-book/master_middleman/source/subnavs/geode-subnav.erb

```diff
@@ -2189,25 +2189,25 @@ gfsh</a>
+                        <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html"><%=vars.product_name%> APIs compatible with Redis</a>
                         <ul>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#using-the-api">Using the Redis API for <%=vars.product_name%></a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html#using-the-api">Using <%=vars.product_name%> APIs Compatible with Redis</a>
                             </li>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#included-commands">Included Redis Commands</a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html#supported-commands">Supported Redis Commands</a>
                             </li>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#advantages-over-redis">Advantages of <%=vars.product_name%> over Redis</a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis#advantages-over-redis">Advantages of <%=vars.product_name%> over Redis</a>
                             </li>
                             <li>
-                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#expiration-accuracy">Expiration Accuracy</a>
+                                <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html#expiration-accuracy">Expiration Accuracy</a>
                             </li>
                              <li>
-                              <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#high-availability-model">High Availability Model</a>
+                              <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html#high-availability-model">High Availability Model</a>
                             </li>
                             <li>
-                              <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#loss-of-connections">Loss of Connections</a>
+                              <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_api_compatible_with_redis.html#loss-of-connections">Loss of Connections</a>
                             </li>
                         </ul>
                     </li>
```

Comment:
I think the file name should be 
geode_apis_compatible_with_redis.html

api needs an 's' at the end of it.